### ISSUE TITLE
The Gradcamplus image becoming Bright

### ISSUE THREAD

**Issue by Arkaprabha1012** [samson6460/tf.keras-gradcamplusplus#3]
The gradcamplus picture after heatmap becomes brighter. Can you please tell me why it is happening in your code? Why it is not resulting like gradcam in the background but with better gradients?

**Comment by samson6460** [comment]
Sorry, I'm not sure about the cause of the problem you met.
I have added a sample code on Colab.
So, you can check it, and maybe you'll find the problem.

**Comment by Arkaprabha1012** [comment]
Thanks a lot samson. Can you please share me the link?

**Comment by samson6460** [comment]
> Thanks a lot samson. Can you please share me the link?

Here you are:
https://colab.research.google.com/drive/1pRlq73Wkd5np3mV-clZOhxmkdnGgpoec?usp=sharing
